src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md

```diff
@@ -3,369 +3,191 @@ path: "/docs/writing-own-tests/setting-up-galasa-project"
 
 ## Before you start
 
-First, you need to ensure that you have installed the Galasa Eclipse plug-in and can run one or more of the tests in an example project. Don't rush this - it is recommended that you familiarise yourself with all of the Galasa examples before creating your own project. But as a minimum, if you can run <a href="/docs/running-simbank-tests/simbank-IVT" target="_blank">The SimBank IVT</a> test, you are all set.
+First, check that you can run one or more of the tests in an example project. Don't rush this - it is recommended that you familiarise yourself with all of the Galasa examples before creating your own project. But as a minimum, if you can run <a href="/docs/running-simbank-tests/simbank-IVT" target="_blank">The SimBank IVT</a> test, you are all set.
 
 ## A little plan
 
-It is convenient to use Eclipse's sub-project feature, so that a full (parent) Galasa project includes several sub-projects, which can also be known as _modules_, some of which are mandatory and some optional.
+A full (parent) Galasa project includes several sub-projects, which can also be known as _modules_, some of which are mandatory and some optional. A parent project can contain the following sub-projects:
 
-We are going to build a hierarchy of projects, where the parent project contains:
-
-- A Managers sub-project, allowing you to extend the provided range of Managers. In practice, if you have no intention of writing a Manager, you can omit this - it is included in the sample project for completeness.
+- A Managers sub-project, allowing you to extend the provided range of Managers. In practice, if you have no intention of writing a Manager, you can omit this.
 - An OBR (OSGi Bundle Repository) sub-project, which is mandatory. Galasa uses the OBR to locate your test project(s) and reason about their interdependencies.
-- One or more test sub-projects, that as the name implies, contain the tests themselves. The sample project will contain just one test project.
+- One or more test sub-projects, that as the name implies, contain the tests themselves. 
 
-The parent project establishes all the dependencies for the sub-projects/modules. It builds all the modules in the order of the dependencies - it builds the Manager module before the test projects that use it.
+The parent project establishes all the dependencies for the sub-projects or modules. It builds all the modules in the order of the dependencies - it builds the Manager module before the test projects that use it.
 
 For simplicity, it is assumed that you will only have one version of a test in production at any one time. However, by establishing different versions of your tests, you can have test streams with different versions of the same test project. For the purposes of the forthcoming example, the version of all projects is set to `0.1.0-SNAPSHOT`. The `SNAPSHOT` element forces Galasa to run with the absolute latest copy of your built test project.
```

Comment:
> The `SNAPSHOT` element forces Galasa to run with the absolute latest copy of your built test project. 

Not really. 0.1.0-SNAPSHOT is just the version which has been set. SNAPSHOT is an indicator that the code is development code, rather than release/production-level.

(Of itself, you could have a version of 0.23.45-AnythingIwantHere, but when someone wants to identify a version of build artifacts they have to know the version number. SNAPSHOT is just a convention, doesnt 'force' anything at all.)

Reply:
I have removed this wording